extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -521,8 +520,12 @@ export class AmpConsent extends AMP.BaseElement {
 
     this.consentConfig_ = consents;
     if (config['postPromptUI']) {
-      this.postPromptUI_ = this.getAmpDoc().getElementById(
```

Comment:
Error message needs to change now it's not required to be a child of amp-consent

Reply:
done